ftw/lawgiver/wdl/parser.py

```diff
@@ -257,3 +260,16 @@ def _get_consumers(self):
                 consumers.append((consumer_constraint, item))
 
         return consumers
+
+    def _get_options(self, line):
+        options = {}
+        match = re.search(r'\[(.*?)\]', line)  # find everything between brackets
+        if not match:
+            return options
+
+        options_string = match.group(1)
+        for option_string in options_string.split('; '):
```

Comment:
`'a=1; b=2'` is supported, but `a=1;b=2` not and `a=1 ; b=2` will probably not have the correct result.
I would change that to `.split(';')` and do a `.strip()` afterwards.